### PR TITLE
chore: upgrade dockertest to v3.8.1

### PR DESCRIPTION
**Pull request by chrikar**
# Description

- Upgrade dockertest to v3.8.1
- lint fixes
- Rearrange imports in changed files in order standard, 3rd-party,
  rudderstack libs

## Notion Ticket

https://www.notion.so/rudderstacks/chore-upgrade-dockertest-to-v3-8-1-e0448643bbb64ee1b6322662132e60bb

## Security

- [x] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

### CODE REVIEW

**Review by atzoum — APPROVED** [review]

File: app/cluster/integration_test.go [hunk 1]
```diff
@@ -249,7 +248,10 @@ func TestDynamicClusterManager(t *testing.T) {
 
 	wait := make(chan bool)
 	go func() {
-		dCM.Run(ctx)
+		err := dCM.Run(ctx)
+		if err != nil {
+			log.Printf("cluster runner stopped: %s \n", err)
```
Comment (fracasula) [inline app/cluster/integration_test.go:253]: Can we use `t.Logf` inside tests please? They respect `testing.Verbose()` and don't require the trailing `\n`.

File: go.mod [hunk 2]
```diff
@@ -71,6 +70,8 @@ require (
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 )
 
+require github.com/ory/dockertest/v3 v3.8.1
```
Comment (fracasula) [inline go.mod:73]: [nitpick] We could have this `require` where it belongs (e.g. up if direct dependency, down if indirect).

File: services/filemanager/fileManager_test.go [hunk 3]
```diff
@@ -19,12 +19,13 @@ import (
 
 	"cloud.google.com/go/storage"
 	"github.com/minio/minio-go/v6"
-	"github.com/ory/dockertest"
+	"github.com/ory/dockertest/v3"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/api/option"
```
Comment (fracasula) [inline services/filemanager/fileManager_test.go:24]: Thanks for sorting these :bow:

File: go.mod (hunk 2 above)
Comment (chrikar) [reply go.mod:73]: This was added automatically during `go install`, but I can put it with the rest of them

File: app/cluster/integration_test.go (hunk 1 above)
Comment (chrikar) [reply app/cluster/integration_test.go:253]: Nice idea! Thx dude!

**Review by fracasula — APPROVED** [review]
LGTM, thank you for cleaning up a bit as well :+1:

**Review by fracasula — APPROVED** [review]

**Pull request merged**